examples/crate_universe/MODULE.bazel

```diff
@@ -345,6 +346,25 @@ use_repo(
     "crate_index_cargo_workspace",
 )
 
+###############################################################################
+# R E M O V E   I N T E R N A L   D E P S
+###############################################################################
+
+# https://bazelbuild.github.io/rules_rust/crate_universe_bzlmod.html
+crate_index_remove_internal_deps = use_extension("@rules_rust//crate_universe:extensions.bzl", "crate")
+crate_index_remove_internal_deps.from_cargo(
```

Comment:
I'm tempted to add a second `from_cargo` with the un-stripped `Cargo.lock`, and a second `number_printer` target pointing at the unstripped repo, so that we can see both `Cargo.lock` files work and are up to date - WDYT?